src/main/scala/chisel3/experimental/util/algorithm/Bitwise.scala

```diff
@@ -0,0 +1,29 @@
+package chisel3.experimental.util.algorithm
```

Comment:
I think putting these kind of hardware algorithm to `algorithm` is a good idea.
 
```suggestion
// SPDX-License-Identifier: Apache-2.0

package chisel3.experimental.util.algorithm
```